### PR TITLE
Add fromDbStringLocal()

### PR DESCRIPTION
**Pull request by ihmc3jn09hk**
Helper function
Maybe operators like +, - shall be added later

### CODE REVIEW

File: trantor/utils/Funcs.h [hunk 1]
```diff
@@ -30,4 +31,16 @@ inline uint64_t ntoh64(uint64_t n)
 {
     return hton64(n);
 }
+inline std::vector<std::string> strsplit(const std::string &s, const std::string &delimiter )
```
Comment (an-tao) [inline trantor/utils/Funcs.h:34]: this is implemented in drogon:
https://github.com/an-tao/drogon/blob/17b8c337f9fe6513e81fa3784bc7559cd7f8fcb7/lib/inc/drogon/utils/Utilities.h#L48-L51

Comment (ihmc3jn09hk) [reply trantor/utils/Funcs.h:34]: I see there is an implementation in drogon. However, trantor is not depended on drogon. Therefore, I reimplemented it and see drogon's splitString wrap on trantor implementation?
Or would it be good to move it to trantor?

Comment (an-tao) [reply trantor/utils/Funcs.h:34]: Oh, I see, this is ok, please use Hump nomenclature. (change strsplit to splitString)

Comment (ihmc3jn09hk) [reply trantor/utils/Funcs.h:34]: I will update that name and wrap the Drogon's functions if needed.

Comment (an-tao) [reply trantor/utils/Funcs.h:34]: Ok! thanks

**Pull request merged**